### PR TITLE
feat: Add delete record

### PR DESCRIPTION
**Pull request by EdwinBetanc0urt**
#### Steps to reproduce

1. Login.
2. Open any window with records.
3. Select action `Delete Record`.

#### Screenshot or Gif

https://user-images.githubusercontent.com/20288327/132793519-4fe40721-7f1c-4929-acec-ada4bb6de8d0.mp4

#### Other relevant information
* Your OS: Linux Mint 19.1 Cinnamon x64. 
* Browser: Mozilla Firefox 88.0.1.
* Node.js version: 14.17.0.
* NPM version: 7.19.0.
* adempiere-vue version: 4.3.1.

#### Additional context
The delete record is pointing to the old service (`/common/api/delete`) that uses the id or uuid of the record and the table name, you should use instead of the table name the uuid of the window and the tab.

### CODE REVIEW

File: src/api/ADempiere/common/persistence.js [hunk 1]
```diff
@@ -162,6 +164,7 @@ export function getEntity({
 
 /**
  * Object List from window
+ * @deprecated replace with /user-interface/persistence/entities
```
Comment (yamelsenih) [inline src/api/ADempiere/common/persistence.js]: Delete it if is deprecate

File: src/components/ADempiere/ActionMenu/Actions.vue [hunk 2]
```diff
@@ -178,56 +180,35 @@ export default defineComponent({
...
-            break
-          case 'recordAccess':
-            icon = 'el-icon-c-scale-to-original'
-            break
-        }
-      }
-
-      return icon
+    const runAction = (action) => {
+      action.callBack({
```
Comment (yamelsenih) [inline src/components/ADempiere/ActionMenu/Actions.vue]: This should be action.runAction function instead callback function, the name should not be callback.

File: src/views/ADempiere/Window/MultiTabWindow.vue [hunk 3]
```diff
@@ -127,43 +126,23 @@ export default defineComponent({
...
-          },
-          deleteRecord,
-          sharedLink
-        ]
-
-        return actionsList
-      }
+      parentUuid: props.windowMetadata.uuid,
+      containerUuid: props.windowMetadata.currentTabUuid,
+      getActionList: () => [
```
Comment (yamelsenih) [inline src/views/ADempiere/Window/MultiTabWindow.vue:131]: Where is implemented the actions list here? 
Where is implemented the actions click here?

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/views/ADempiere/Window/MultiTabWindow.vue (hunk 3 above)
Comment (EdwinBetanc0urt) [reply src/views/ADempiere/Window/MultiTabWindow.vue:131]: this returns the list of actions, they are imported as individual actions according to the implementation and added to the method

File: src/components/ADempiere/ActionMenu/Actions.vue (hunk 2 above)
Comment (EdwinBetanc0urt) [reply src/components/ADempiere/ActionMenu/Actions.vue]: is a list of objects, where each action has a name, an icon and a callBack, and the name of the callBack method could be changed, but that same name must be stored in another property, leaving something similar to:
```javascript
const runAction = (action) => {
  action[actionName]({

  })
}
```

File: src/api/ADempiere/common/persistence.js (hunk 1 above)
Comment (EdwinBetanc0urt) [reply src/api/ADempiere/common/persistence.js]: depreciated `getEntities` implementations were removed

File: src/components/ADempiere/ActionMenu/Actions.vue (hunk 2 above)
Comment (yamelsenih) [reply src/components/ADempiere/ActionMenu/Actions.vue]: just change the name to action

Comment (EdwinBetanc0urt) [reply src/components/ADempiere/ActionMenu/Actions.vue]: changes implemented

**Pull request merged**